docker-entrypoint.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+function stop {
+	kill $(ps -C ts3server -o pid= | awk '{ print $1; }')
+	exit
+}
+
+trap stop INT
+trap stop TERM
+
+test -d /data/files || mkdir -p /data/files && chown teamspeak:teamspeak /data/files
+ln -sf /data/files $TS_DIRECTORY/files
+
+test -d /data/logs || mkdir -p /data/logs && chown teamspeak:teamspeak /data/logs
+ln -sf /data/logs $TS_DIRECTORY/logs
+
+ln -sf /data/query_ip_whitelist.txt $TS_DIRECTORY/query_ip_whitelist.txt
```

Comment:
My reasoning with the for loop was that if there is only the static list of files that can be stored in the volume and symlinked, you would have to modify the entrypoint script to add any other files. Doing it with a for loop and linking everything at startup, you allow users to drop in any files they wish and they will automatically be symlinked. I don't know if there are other files that TS can use besides those symlinked in the entrypoint but this would add that flexibility. It doesn't change that they're symlinked so it has the same intent to allow for persistent data and non-persistent data to coexist. The broke symlink cleanup was just replacing the license link cleanup with something that would clean up all symlinks. Figured might as well go all out if it was going to be done.